packages/sharec-docs/content/_index.md

```diff
@@ -10,7 +10,7 @@ keep your code up to date and start new projects in one command.
 
 ## Quick start
 
-<img src="public/logo-alt.svg" alt="Sharec logo by Ivashkina Xenia <xeniaowl112@mail.ru>" align="right" width="100">
+<img src="https://raw.githubusercontent.com/lamartire/sharec/09bdf916b6d553dbe57d686154daafd3de0d2979/public/logo-alt.svg" alt="Sharec logo by Ivashkina Xenia <xeniaowl112@mail.ru>" align="right" width="100">
```

Comment:
Are you shure, that it will not be broken after next commit?

Reply:
I think it is better to move all logos and other static files to `docs` repo and then load them from `sharec.js.org`. What do you think?

Reply:
Ya the image change had me uncertain.  Moving them into the docs repo is probably the best idea. You currently have the one logo hosted from `https://sharec.js.org/images/logo.svg`   which is the themes image folder `/sharec-docs/themes/sharec/static/images/`.

Looking at the [Hugo static file](https://gohugo.io/content-management/static-files/) docs it seems like it would be best to just move them to the static folder as the images should be independent of the theme.

```suggestion
<img src="https://sharec.js.org/images/logo-alt.svg" alt="Sharec logo by Ivashkina Xenia <xeniaowl112@mail.ru>" align="right" width="100">
```

Without having the build tools installed I can't really be certain my suggested changes will break the links.

Reply:
I created a new commit that I believe solves this moving the `public` image folder to the static folder.